src/components/article copy.jsx

```diff
@@ -0,0 +1,39 @@
+import {useParams} from 'react-router-dom'
+import {useState, useEffect} from 'react'
+import {getArticleByArticleId} from '../utils/api'
+
+
+function Article(article) {
+    const { article_id } = useParams();
+    const postedAt = article.created_at.slice(0, article.created_at.length -5).replace('T', '  ')
```

Comment:
This could be done 100x easier by declaring it as a date, then using a formatting method like toUTCstring() for example!